recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 1af9c3bb6f07f6266aeb038fef5a1fa63dafb70a466dcdfe44b6627e8d9aec59
 
 build:
-  number: 0
+  number: 1
   noarch: python
```

Comment:
noarch should only be triggering a linux build, not sure what's going on here...